plugins/SitesManager/tests/Integration/ApiTest.php

```diff
@@ -547,11 +555,13 @@ public function test_getSitesWithAdminAccess_shouldOnlyReturnSitesHavingActually
     {
         API::getInstance()->addSite("site1", array("http://piwik.net", "http://piwik.com/test/"));
         API::getInstance()->addSite("site2", array("http://piwik.com/test/"));
-        API::getInstance()->addSite("site3", array("http://piwik.org"));
+        API::getInstance()->addSite("site3", array("http://piwik.org"), null, null, null, null, null, null, 'Asia/Tokyo');
+
+        Translate::loadAllTranslations();
 
         $resultWanted = array(
-            0 => array("idsite" => 1, "name" => "site1", "main_url" => "http://piwik.net", "ecommerce" => 0, "excluded_ips" => "", 'sitesearch' => 1, 'sitesearch_keyword_parameters' => '', 'sitesearch_category_parameters' => '', 'excluded_parameters' => '', 'excluded_user_agents' => '', 'timezone' => 'UTC', 'currency' => 'USD', 'group' => '', 'keep_url_fragment' => 0, 'type' => 'website', 'exclude_unknown_urls' => 0),
-            1 => array("idsite" => 3, "name" => "site3", "main_url" => "http://piwik.org", "ecommerce" => 0, "excluded_ips" => "", 'sitesearch' => 1, 'sitesearch_keyword_parameters' => '', 'sitesearch_category_parameters' => '', 'excluded_parameters' => '', 'excluded_user_agents' => '', 'timezone' => 'UTC', 'currency' => 'USD', 'group' => '', 'keep_url_fragment' => 0, 'type' => 'website', 'exclude_unknown_urls' => 0),
+            0 => array("idsite" => 1, "name" => "site1", "main_url" => "http://piwik.net", "ecommerce" => 0, "excluded_ips" => "", 'sitesearch' => 1, 'sitesearch_keyword_parameters' => '', 'sitesearch_category_parameters' => '', 'excluded_parameters' => '', 'excluded_user_agents' => '', 'timezone' => 'UTC', 'timezone_name' => 'GMT', 'currency' => 'USD', 'group' => '', 'keep_url_fragment' => 0, 'type' => 'website', 'exclude_unknown_urls' => 0),
```

Comment:
The timezone name for `UTC` should be `UTC` and not `GMT`, right?

Reply:
Well, CLDR uses GMT. This translation is used in most locales. I don't have much opinion on the specific translation, but I generally think we should follow CLDR (a high-quality source backed by heavy industry giants) unless there are strong arguments against it.

Reply:
As an astronomy pedantic I have to mention that UTC and GMT are not the same, even though they are quite similar:

> For most purposes, UTC is considered interchangeable with Greenwich Mean Time (GMT) but GMT is no longer precisely defined by the scientific community.
https://en.wikipedia.org/wiki/Coordinated_Universal_Time

But to be fair the differences don't really matter, even though I think most people expect reading UTC in an IT-context.

Reply:
agreed, it's important to write "UTC" everywhere (and "GMT" should not be visible anywhere) :+1:

Reply:
If there is consensus that we want this, I'll skip importing translations for “UTC” from CLDR and fetch them from Transifex instead. However, I would much prefer if we could outsource such decisions to CLDR, so we can focus on matters specific to web analytics.

Even though I tend to agree that UTC is more prevalent these days and is technically more correct, I would say that both are acceptable. E.g. Google Analytics uses GMT. 

FWIW, Microsoft used GMT in previous versions of Windows, but now they use UTC, at least in the English version. Microsoft are active contributors to CLDR, but it seems they have chosen to deviate from the CLDR data her. They may wish to support a proposal to change the CLDR translation.

Reply:
Yes @c960657 we want to see UTC everywhere (and nowhere GMT)